action.yml

```diff
@@ -17,10 +17,6 @@ inputs:
     default: https://api.mergify.com/v1
   name:
     description: Name of the current workflow job
-outputs:
-  CI_ISSUE_GIGID:
```

Comment:
It is used in the CI check still

Reply:
Can you be more precise?

Reply:
yes, the workflow `ci.yaml` is using the output for its test.

Reply:
I changed the CI to check what matters, the annotations.